src/ca65/instr.c

```diff
@@ -373,6 +381,153 @@ static const struct {
     }
 };
 
+/* Instruction table for the Rockwell C39 */
+static const struct {
+  unsigned Count;
+  InsDesc  Ins[135]; // FIX WHEN ADDING INSN!
+} InsTabC39 = {
+    sizeof (InsTabC39.Ins) / sizeof (InsTabC39.Ins[0]),
+    {
+      // C39: (ind)   = AM65_DIR_IND   = 0x0400 allowed
+      // C39: (ind,x) = AM65_DIR_X_IND = 0x8000 not allowed
+      { "ADC",  0x080266C, 0x60, 0, C39PutArith }, 
+      { "ADX",  0x0800024, 0x80, 0, C39PutAdd }, // C39: renamed ADD to ADX because it conflicts with add macro
```

Comment:
I thought about renaming the 'add' macro and replacing all occurrences of it in the runtime lib. But it*s probably used outside the runtime lib in user programs, so it cannot be renamed.

So using a different name for the "ADD" instruction is probably the only way to go.